src/msr_data_arch.cpp

```diff
@@ -966,6 +966,20 @@ namespace geopm
                 }
             }
         },
+        "IA32_ENERGY_PERF_BIAS": {
```

Comment:
We usually leave the IA32_ part off the name of these

Reply:
Removed in the newest patch.